pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -102,7 +102,7 @@ func (pr *PipelineRun) PipelineTimeout(ctx context.Context) time.Duration {
 	if pr.Spec.Timeouts != nil && pr.Spec.Timeouts.Pipeline != nil {
 		return pr.Spec.Timeouts.Pipeline.Duration
 	}
-	return time.Duration(config.FromContextOrDefaults(ctx).Defaults.DefaultTimeoutMinutes) * time.Minute
+	return time.Duration(apisconfig.FromContextOrDefaults(ctx).Defaults.DefaultTimeoutMinutes) * time.Minute
```

Comment:
looks like this change in `PipelineTimeout` is because of the refactor removing the duplicated import, can we move this cleanup to its own PR?

Reply:
@jerop thanks for the review, yeah I'll revert this